libs/policy/policy.go

```
@@ -292,22 +391,28 @@ func (p DiggerPolicyChecker) CheckAccessPolicy(ciService ci.OrgService, prServic
+
 	query, err := rego.New(
 		rego.Query("data.digger.allow"),
 		rego.Module("digger", policy),
 	).PrepareForEval(ctx)
 
 	if err != nil {
+		slog.Error("Failed to prepare policy evaluation", "error", err)
 		return false, err
 	}
 
 	results, err := query.Eval(ctx, rego.EvalInput(input))
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)